src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -50,36 +50,71 @@ class FeatureFlagValue:
         Represents if the Feature flag is On/Off/Conditionally On
     :ivar dict {string, FeatureFilter[]} conditions:
         Dictionary that contains client_filters List (and server_filters List in future)
-    '''
-
-    def __init__(self,
-                 id_,
-                 description=None,
-                 enabled=None,
-                 conditions=None,
-                 display_name=None):
+    :ivar FeatureAllocation allocation:
+        Determines how variants should be allocated for the feature to various users.
+    :ivar list FeatureVariant[] variants:
```

Comment:
Is this to signify a list of lists?

Reply:
No should be a list. This has been updated